lz4.opam

```
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+name: "lz4"
+version: "1.1.1"
+synopsis: "Bindings to the LZ4 compression algorithm"
+maintainer: "whitequark <whitequark@whitequark.org>"
+authors: [ "whitequark <whitequark@whitequark.org>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/whitequark/ocaml-lz4"
+doc: "http://whitequark.github.io/ocaml-lz4"
+bug-reports: "https://github.com/whitequark/ocaml-lz4/issues"
+dev-repo: "git://github.com/whitequark/ocaml-lz4.git"
+tags: [ "lz4" "compression" "decompression" ]
+build: [
+  ["dune" "build" "@install" "-j" jobs "-p" name]
+  ["dune" "runtest" "-j" jobs "-p" name] {with-test}
+  ["dune" "build" "@doc" "-j" jobs "-p" name] {with-doc}
+]
+depends: [
+  "base-bytes"
```

`base-bytes` isn’t needed (`bytes` is used nowhere) and pulls `ocamlfind` for backward-compatibility purpose
```suggestion
  "ocaml" {>= "4.02"}
```